src/segment-loader.js

```diff
@@ -1905,6 +1907,14 @@ export default class SegmentLoader extends videojs.EventTarget {
     return true;
   }
 
+  getCurrentMediaInfo_(segmentInfo = this.pendingSegment_) {
+    return segmentInfo && segmentInfo.trackInfo || this.currentMediaInfo_;
```

Comment:
Am I reading this right that basically `currentMediaInfo_` is the trackInfo, but that there's a sync issue between what the SegmentLoader has (or doesn't have) and what the actual segmentInfo is. Does it make sense to even have `currentMediaInfo_` at all? Should we remove it and rely only on segmentInfo.trackInfo?

Reply:
I was wondering the same myself during the creation of this pull request but was running into issues in a few areas trying to get segmentInfo.trackInfo working across the board. I think it's likely that we should do this in the future, but for now we should stick with this. I will add a task to #1057 to think about why we need current/starting media info and wether we should just be using `segmentInfo.trackInfo`.

Reply:
Sounds good.